packages/pdf-api/src/draft/draft.controller.ts

```diff
@@ -37,33 +38,58 @@ export class DraftController {
   })
   @ApiBody({
     description:
-      'Arbitrary JSON draft parameters. This can be any JSON object.',
+      'Draft data with optional newName for renaming or params for updating content.',
```

Comment:
Interesting decision :) Would like to see this as part of technical design, e.g. something like "allow renaming drafts by passing optional new name in PUT request".